src/receive_imf.rs

```diff
@@ -842,6 +842,7 @@ async fn add_parts(
             }
         }
 
+        // Lookup or create adhoc group chat.
```

Comment:
`lookup_chat_or_create_adhoc_group()` may look up not only ad-hoc group chat. Is some comment really needed here?